aws-samples/awsome-distributed-training
# chore: Update NCCL version, add aws-ofi-nccl tuner plugin

*Issue #, if available:*

*Description of changes:*
- Update NCCL to latest version
- Remove `FI_PROVIDER`, `FI_EFA_USE_DEVICE_RDMA`, and `FI_EFA_FORK_SAFE` environment variables per https://github.com/aws/aws-ofi-nccl/blob/master/doc/efa-env-var.md given the aws-ofi-nccl version used
- Add support for using the aws-ofi-nccl tuner plugin (`NCCL_TUNER_PLUGIN`) 
- Move environment variables up to `env` section and reference in `command` accordingly

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
